core/commands/storage.go

```diff
@@ -250,7 +250,7 @@ Receive proofs as collateral evidence after selected nodes agree to store the fi
 			ss.GetOrDefault(singleChunk)
 		}
 		testFlag := req.Options[testOnlyOptionName].(bool)
-		go retryMonitor(context.Background(), api, ss, n, ssID, testFlag)
+		go retryMonitor(req.Context, api, ss, n, ssID, testFlag)
```

Comment:
Please keep `context.Background()`. Reason is that once this command finishes, `req.Context` is freed, so the goroutine can never continue using this context (it's cancelled automatically, making all calls using context exit immediately).